src/app/index.js

```diff
@@ -69,16 +69,22 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
       return $delegate;
     });
   })
-  .run(function($http, jhConfig) {
+  .run(function($rootScope,$http, jhConfig) {
+    //global function to replace the placeholders
+    $rootScope.replacePlaceholder = function(value){
+      return value.replace("%{hostname}",window.location.host)
```

Comment:
Please, observe style rules:

* Use a space to separate arguments: `%{hostname},window` to `%{hostname}, window`.
* Uses semicolon at the end of the line.

You can check the output of `gulp scripts` to find out style problems. On the other hand, when PR https://github.com/jangouts/jangouts/pull/178 is merged, you'll be able to run `gulp lint` to check your code style.